### PR TITLE
Left mouse button click on any element completed.

### PR DESCRIPTION
**Pull request by robert-duda-cap**
Ready for code review.

### CODE REVIEW

File: allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/BasePage.java [hunk 1]
```diff
@@ -353,17 +353,29 @@ public static void openInNewTab(String url) {
 		WindowUtils.switchWindow(url, true);
 	}
 	
+	public static void performLeftMouseClickOnWebElement(By selector) {
+		WebElement element = getDriver().findElementQuietly(selector);
+		if (element != null)
+			performLeftMouseClickOnWebElement(element);
```
Comment (lstefaniszyn) [inline allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/BasePage.java]: Please inform user that "performLeft...." action was not executed due to ....

File: allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/BasePage.java [hunk 2]
```diff
@@ -353,17 +353,29 @@ public static void openInNewTab(String url) {
 		WindowUtils.switchWindow(url, true);
 	}
 	
+	public static void performLeftMouseClickOnWebElement(By selector) {
```
Comment (lstefaniszyn) [inline allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/BasePage.java]: Please move them to DriverExtention.class, where we keep all actions on driver

**Comment by lstefaniszyn** [comment]
@robert-duda-cap :  Is this PR connected with "horizontal slider test completed #69" ? Can it be removed, while left click was merge there

**Comment by robert-duda-cap** [comment]
I am removing PR as task is already done in another PR#69

**Pull request closed without merging**